manifests/boolean.pp

```diff
@@ -36,8 +36,11 @@
     default              => undef,
   }
 
-  selboolean { $name:
-    value      => $value,
-    persistent => $persistent,
+  # Do nothing unless SELinux is enabled
+  if $facts['selinux'] {
```

Comment:
Isn't the `selinux` [fact](https://puppet.com/docs/facter/3.9/core_facts.html) a hash though (thus always true)?
> selinux (map) — Represents information about Security-Enhanced Linux (SELinux).
config_mode (string) — The configured SELinux mode.
config_policy (string) — The configured SELinux policy.
current_mode (string) — The current SELinux mode.
enabled (boolean) — True if SELinux is enabled or false if not.
enforced (boolean) — True if SELinux policy is enforced or false if not.
policy_version (string) — The version of the SELinux policy.

Reply:
Sorry, my mistake.  `selinux` *is* a boolean legacy fact.  The new fact is `os['selinux']`

Reply:
Which you've already covered in great detail in this [thread](https://github.com/voxpupuli/puppet-selinux/issues/286#issuecomment-512356998).  I'll get my coat... :)